sdks/python/apache_beam/runners/worker/worker_id_interceptor.py

```diff
@@ -39,8 +39,8 @@ class WorkerIdInterceptor(grpc.StreamStreamClientInterceptor):
   # and throw exception in worker_id_interceptor.py after we have rolled out
   # the corresponding container changes.
   # Unique worker Id for this worker.
-  _worker_id = os.environ['WORKER_ID'] if os.environ.has_key(
-      'WORKER_ID') else str(uuid.uuid4())
+  _worker_id = os.environ['WORKER_ID'] if 'WORKER_ID' in os.environ else \
+      str(uuid.uuid4())
```

Comment:
OUCH...  Backshash is a bad idea (see PEP8).  One space character to the right of the backslach and the script breaks on a change that is not visible to the reader.  What about ___worker_id = os.environ.get('WORKER_ID', str(uuid.uuid4()))__ instead?

Reply:
+1.